akka-http-core/src/main/scala/akka/http/model/japi/Http.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.http.model.japi
```

Comment:
We don't have the packages migrated, have you seen our proposal how to name things in #17279 ? We wanted to wait until things are merged before we introduce an all-conflicting renaming.